applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py

```diff
@@ -92,7 +92,9 @@ def GetDefaultSettings(cls):
             "analysis_type": "non_linear",
             "model_import_settings": {
                 "input_type": "mdpa",
-                "input_filename": "unknown_name"
+                "input_filename": "unknown_name",
+                "physics_input_type": "ph.json",
+                "physics_file_name": "physics"
```

Comment:
please remove those, the parameters are not validated recursively, hence this is not used in any case

Reply:
Why shouldn't they be here. I can't add things to the parameters of they're not here or was this changed recently?

Reply:
my thinking is that 
1. those are never used, since the import_settings have to be provided in any case
2. different methods of imports have different settings. So either we add all of them or none. Adding all of them is IMO more confusing, since it will be many (e.g restart related, for mpi, ...)

Reply:
That means I can just add whatever I want?